res/bnd/catalina.jar.tmp.bnd

```diff
@@ -55,15 +55,15 @@ Export-Package: \
         access=32;\
         version=${Bundle-Version};\
         modules='\
-            jakarta.ejb-api,\
-            jakarta.mail,\
-            jakarta.persistence-api,\
-            jakarta.xml.rpc-api,\
-            java.xml.ws'
+            jakarta.ejb.api,\
+            jakarta.mail.api,\
+            jakarta.persistence.api,\
+            jakarta.xml.rpc.api,\
```

Comment:
XML RPC has been dropped in Jakarta EE 9. We'll need to clean that up in Tomcat. I guess this one can stay as is for now.